### PR TITLE
Add a game with a kitty 3

### PR DESCRIPTION
**Pull request by marjuanm**
A game with a kitty is a platform game similar to the early Mario Bros games, Kirby and other games. Works fine on Reactos (sound included).

### CODE REVIEW

**Comment by JoachimHenze** [comment]
The Vectorian Giotto changes in here seem unrelated. The rapps-db-validation-check is failing as well.

**Comment by marjuanm** [comment]
[agwak.txt](https://github.com/reactos/rapps-db/files/14230051/agwak.txt)
Sorry, 

> The Vectorian Giotto changes in here seem unrelated. The rapps-db-validation-check is failing as well.



> The Vectorian Giotto changes in here seem unrelated. The rapps-db-validation-check is failing as well.

Sorry, I'm still a newbie to Github and I still have a lot of problems creating new pull requests.

File: agwak.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Section]
+Name = A game with a kitty 3
+Version = 3.0.0
+License = Freeware
+LicenseType = 2
+Description = A game with a kitty is a platform game with a dynamic similar to the first Mario Bros, Kirby and another games. We must go through a series of missions jumping between platforms and evading various enemies.
```
Comment (JoachimHenze) [inline agwak.txt:6]:
```suggestion
Description = A platform game similar to Mario Bros, Kirby and other games. We must go through a series of missions jumping between platforms and evading various enemies.

```

File: agwak.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[Section]
+Name = A game with a kitty 3
+Version = 3.0.0
+License = Freeware
+LicenseType = 2
+Description = A game with a kitty is a platform game with a dynamic similar to the first Mario Bros, Kirby and another games. We must go through a series of missions jumping between platforms and evading various enemies.
+Category = 4
+URLSite = https://www.origamihero.com/ori6/games.php
+URLDownload = http://www.origamihero.com/games/kitty3.zip
```
Comment (JoachimHenze) [inline agwak.txt:9]:
```suggestion
URLDownload = https://www.origamihero.com/games/kitty3.zip

```

**Comment by marjuanm** [comment]
Help!!!!!, I can't find how to remove the vectorian giotto references from this request, I'm trying to create the request for a game with a kitty, the correct files are:
[agwak.txt](https://github.com/reactos/rapps-db/files/14230109/agwak.txt)
icon
Sorry, I dont upload icons from here :(

**Comment by JoachimHenze** [comment]
It is possible to just close this PR and open a new one as a replacement. No problem.

**Comment by marjuanm** [comment]
> Es posible simplemente cerrar este PR y abrir uno nuevo como reemplazo. Ningún problema.

ok, thanks, I should this PR then?

**Pull request closed without merging**